src/workflows/deploy.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+GIT_BRANCH="${1:-""}"
+
+fail_if_unset () {
+  VAR_NAME="$1"
+  if [[ -z "${!VAR_NAME:-""}" ]]; then
+    printf '%s not set\n' "$VAR_NAME" >&2
+    exit 1
+  fi
+}
+gwei_to_wei () {
+  GWEI="$1"
+  printf "%s000000000" "${GWEI}"
+}
```

Comment:
At first it looked like `GWEI` was fixed at 1, but then I saw the $

Reply:
This is, unfortunately, how you pass parameters to functions in Bash. Typically you achieve "named parameters" just how its done here:
```bash
my_function () {
  named_parameter="$1"
  second_named_parameter="$2"
  # ...
  
  # Do the stuff here with named parameters
}
```